### DISCUSSION TITLE
Why aren't mappable actions and remote control actions unified?

### DISCUSSION THREAD

**Discussion by hacker-DOM** [kovidgoyal/kitty#6794] [General]
There's a lot of overlap between [the](https://sw.kovidgoyal.net/kitty/actions/) [two](https://sw.kovidgoyal.net/kitty/remote-control). The only fundamental difference I can see is that remote control action have `--match` to find a window 

If that's the case, is there a case for a standardized api (with `--match` in remote control)

**Comment by kovidgoyal** [comment]
https://sw.kovidgoyal.net/kitty/remote-control/#mapping-key-presses-to-remote-control-commands
